index.js

```diff
@@ -11,9 +11,11 @@ module.exports = function (content, map, meta) {
   }
 
   var callback = this.async()
-  require('imagemin')
-    .buffer(content, options)
-    .then(function (buffer) { callback(null, buffer) })
+  eval('import("imagemin")')
```

Comment:
why `eval('import("imagemin")')` over just doing `import('imagemin')`?

Reply:
`eslint` was complaining in my IDE:
```
Parsing error: The keyword 'import' is reserved
```

I had seen other libs using the `eval` tip and I thought I'd follow it. Happy to go either way

Reply:
I'd rather not use `eval`. There may be something you can specify in your linter or in package.json to indicate this runs where `import()` is already supported, since imagemin 8 requires node 12 anyway.

Reply:
No pb, I can deal with `eslint` on my side indeed.

I've sent you a separate PR to change Node requirements.